lib/akamod/set_prop.py

```diff
@@ -81,7 +81,8 @@ def unset_prop(body, ctype, prop=None, condition=None, condition_prop=None):
         "mwdl_exclude": lambda v: (v[0] == "collections" or
                                    v[0] == "findingAids"),
         "hathi_exclude": lambda v: "Minnesota Digital Library" in v,
-        "finding_aid_title": lambda v: v[0].startswith("Finding Aid")
+        "finding_aid_title": lambda v: v[0].startswith("Finding Aid"),
+        "usc_no_contributor": lambda v: not v[0].get("contributor", False)
```

Comment:
`getprop()` also takes a final third Boolean argument to determine whether to handle missing properties as `KeyError`s:

```python
>>> from dplaingestion.selector import getprop
>>> import json
>>> data = json.loads('{"sourceResource":{"a":"aaa"}}')
>>> getprop(data, 'sourceResource/b', False)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/v1/ingestion/lib/python2.7/site-packages/dplaingestion/selector.py", line 23, in getprop
    return getprop(obj[pp],pn,keyErrorAsNone)
  File "/v1/ingestion/lib/python2.7/site-packages/dplaingestion/selector.py", line 14, in getprop
    return obj[path]
KeyError: 'b'
>>> getprop(data, 'sourceResource/b', True)
>>> print getprop(data, 'sourceResource/b', True)
None
```

... that said, I'm not sure what the side effects of changing that would be.